pkg/autodiscovery/scheduler/meta.go

```diff
@@ -0,0 +1,49 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package scheduler
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/autodiscovery/integration"
+)
+
+// MetaScheduler is a scheduler dispatching to all registered schedulers
+type MetaScheduler struct{}
+
+// Start inits the meta scheduler
+func (s *MetaScheduler) Start() {
+	// no op
+}
+
+// Schedule schedules configs to all registered schedulers
+func (s *MetaScheduler) Schedule(configs []integration.Config) {
```

Comment:
We're only sending diffs to the schedulers, that means we must make sure all relevant schedulers are listening before we call `StartPolling()`, right?

Reply:
Yes, ping @ajacquemot would that be a potential issue?

Reply:
We just need to register at the same place as the check scheduler, a replay would be nice though.